src/containers/singleEvent/index.tsx

```diff
@@ -77,6 +120,14 @@ const SingleEvent: React.FC<SingleEventProps> = ({ events }) => {
         />
       </Helmet>
       <ContentContainer>
+        {privilegeLevel === PrivilegeLevel.ADMIN ? (
+          <AdminActionButtonList>
+            <GreenButton>Edit</GreenButton>
```

Comment:
⛏️ I would prefer if we made these LinkButtons instead of buttons so that its easier for the next person to just pass the `to` props on these buttons.